pinc/DPage.inc

```diff
@@ -501,7 +501,7 @@ function _Page_require(
             . "    "
             . sprintf(_("But it is in %s"), $curr_page_state)
             ;
-        abort($err);
+        throw new Exception($err);
```

Comment:
We should define and throw/catch a more specific exception for project page errors in all these places. Perhaps a `ProjectPageException`? For now I would probably put this in `Project.inc`.

Reply:
Also this removes all use of `abort()` in the file so we can remove the include().

Reply:
Done these.